client/src/components/ExpensesForm.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+const ExpensesForm = ({ categories }) => {
+  const [date, setDate] = React.useState("");
+  const [amount, setAmount] = React.useState("");
+  const [category, setCategory] = React.useState("");
+  const [desc, setDesc] = React.useState("");
+  const [account, setAccount] = React.useState("");
+
+  return (
+    <div>
+      <h2>Add an expense:</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>Date</th>
+            <th>Amount ($)</th>
+            <th>Category</th>
+            <th>Description</th>
+            <th>Account</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <input name="date" type="date" value={date} onChange={e => setDate(e.target.value)} />
+            </td>
+            <td>
+              <input name="amount" type="number" value={amount} onChange={e => setAmount(e.target.value)} />
```

Comment:
At what point are we going to convert/clean dollar inputs to integer?

Reply:
We will do this in the submission handler for the form